arch/arm/neon_intrins.h

```diff
@@ -1,8 +1,14 @@
 #ifndef ARM_NEON_INTRINS_H
 #define ARM_NEON_INTRINS_H
 
-#if defined(_M_ARM64) || defined(_M_ARM64EC)
-#  include <arm64_neon.h>
+#if (defined(_M_ARM64) || defined(_M_ARM64EC))
+#  if defined(__MINGW32__) && defined(__MINGW64__)
+/* llvm-mingw does not have arm64_neon.h */
+#     include <arm_neon.h>
+#  else
+/* In MSVC, including arm64_neon.h is necessary */
+#     include <arm64_neon.h>
+#  endif
```

Comment:
> Since `arm64_neon.h` is specific to MSVC, it might be better to check for that specifically.

Thanks for advising. I have applied your suggestion in the latest commit.